docs/src/api/class-browsertype.md

```diff
@@ -93,19 +93,16 @@ browser = playwright.chromium.launch( # or "firefox" or "webkit".
 )
 ```
 
-> **Chromium-only** Playwright can also be used to control the Chrome browser, but it works best with the version of
+> **Chromium-only** Playwright can also be used to control the Chrome or Edge browsers, but it works best with the version of
```

Comment:
```suggestion
> **Chromium-only** Playwright can also be used to control the Google Chrome or Microsoft Edge browsers, but it works best with the version of
```
I think using Microsoft Edge is better instead of just Edge.

Reply:
Agreed, thanks!